<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>901</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="na">
     <attribute name="title">
      <string>NA-Mode</string>
     </attribute>
     <widget class="QwtPlot" name="qwtPlot_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>AutoScale</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="phase">
     <attribute name="title">
      <string>Phase</string>
     </attribute>
     <widget class="QwtPlot" name="phasePlot">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frameNA">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>0</y>
        <width>261</width>
        <height>301</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="nnnn">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>59</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>123</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btnPhase">
          <property name="text">
           <string>Load Phase Trace</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btnMeasurePh">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>89</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Measure</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnFFT">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>FFT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnIFFT">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>IFFT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPrueba">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>89</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>PRUEBA **</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnClear">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>320</y>
        <width>89</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Plot</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QwtPlot" name="qwtPlot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>641</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="Frecuency">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>160</width>
       <height>114</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignJustify|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="QLinitfreq">
        <property name="text">
         <string>IInicio</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="QlendFreq">
        <property name="text">
         <string>Final</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="endFreq">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="iniFreq">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="setFreq">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>89</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="freqRange">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frecuencia</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="power">
       <property name="text">
        <string>Potencia</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="powerStep">
       <property name="minimum">
        <number>-45</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>-15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnpwstep">
       <property name="text">
        <string>set</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>300</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnclose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPlainTextEdit" name="log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>431</width>
       <height>111</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CN">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
